dspace/config/crosswalks/oai/metadataFormats/oai_openaire.xsl

```diff
@@ -0,0 +1,1484 @@
+            </xsl:when>
+            <xsl:when test="$lc_identifier_type = 'lissn'">
+                <xsl:text>LISSN</xsl:text>
+            </xsl:when>
+            <xsl:when test="$lc_identifier_type = 'lsid'">
+                <xsl:text>LSID</xsl:text>
+            </xsl:when>
+            <xsl:when test="$lc_identifier_type = 'pmid'">
+                <xsl:text>PMID</xsl:text>
+            </xsl:when>
+            <xsl:when test="$lc_identifier_type = 'purl'">
+                <xsl:text>PURL</xsl:text>
+            </xsl:when>
+            <xsl:when test="$lc_identifier_type = 'upc'">
+                <xsl:text>UPC</xsl:text>
+            </xsl:when>
+            <xsl:when test="$isURL = 'true' or $lc_identifier_type = 'url'">
+                <xsl:text>URL</xsl:text>
+            </xsl:when>
+            <xsl:otherwise>
+                <xsl:text>URN</xsl:text>
+            </xsl:otherwise>
+        </xsl:choose>
+    </xsl:template>
+
+
+
+   <!--  -->
+   <!-- Auxiliary templates - get global values -->
+   <!--  -->
+   
+    <!-- get the coar access rights globally -->
+    <xsl:template name="getRightsURI">
+        <xsl:call-template name="resolveRightsURI">
+            <xsl:with-param name="field"
+                select="//doc:element[@name='dc']/doc:element[@name='rights']/doc:element/doc:field[@name='value']/text()"/>
+        </xsl:call-template>
+    </xsl:template>
+
+   <!-- get the issued date globally -->
+    <xsl:template name="getIssuedDate">
+        <xsl:value-of
+            select="//doc:element[@name='date']/doc:element[@name='issued']/doc:element/doc:field[@name='value']/text()"/>
+    </xsl:template>
+
+   <!-- get the repository baseUrl globally -->
+    <xsl:template name="getDSpaceURL">
+        <xsl:value-of select="//doc:element[@name='repository']/doc:field[@name='url']/text()"/>
+    </xsl:template>
+
+   <!--  -->
+   <!-- Auxiliary templates - dealing with Entities -->
+   <!--  -->
+
+    <!-- 
+        this template will verify if a field name with "authority"
+        is present and if it start with "virtual::"
+        if it occurs, than we are in a presence of an related entity 
+     -->
+    <xsl:template name="isRelatedEntity">
+        <xsl:param name="element"/>
+        <xsl:variable name="sibling1" select="$element/following-sibling::*[1]"/>
+        <!-- if next sibling is authority and starts with virtual:: -->
+        <xsl:choose>
+            <xsl:when test="$sibling1[@name='authority' and starts-with(text(),'virtual::')]">
+                <xsl:value-of select="true()"/>
+            </xsl:when>
+            <xsl:otherwise>
+                <xsl:value-of select="false()"/>
+            </xsl:otherwise>
+        </xsl:choose>
+    </xsl:template>
+
+    <!-- 
+        this template will try to look for all "virtual::"
+        and rebuild an "Entity" based on all virtual fields 
+        in the source data.
+        This will retrieve something like 
```

Comment:
This final line of the comment seems incomplete? It seems like it is meant to have an example?

Reply:
True! I intend to add an example and I forgot to do it :). Just added one.